charts/values.yaml

```diff
@@ -0,0 +1,162 @@
+fullnameOverride: ""
+nameOverride: ""
+
+ingress:
+  enabled: false
+  annotations:
+    kubernetes.io/ingress.class: nginx
+
+secrets:
+  logEncryptionKey: null
+  asyncEncryptionKey: null
+
+dbMigration:
+  enabled: true
+
+controller:
+  controllersRoleArn: arn:aws:iam::ftl-controllers-irsa-role
+  replicas: 2
+  revisionHistoryLimit: 0
+
+  image:
+    repository: "ftl0/ftl-controller"
+    tag: "v0.361.2"
+
+  envFrom: null
+  dbConnectionString: "postgres://$(endpoint):$(port)/tbd?sslmode=disable&user=$(username)&password=$(password)"
+  kmsUri: "fake-kms://CKbvh_ILElQKSAowdHlwZS5nb29nbGVhcGlzLmNvbS9nb29nbGUuY3J5cHRvLnRpbmsuQWVzR2NtS2V5EhIaEE6tD2yE5AWYOirhmkY-r3sYARABGKbvh_ILIAE"
```

Comment:
This will result in the same fake KMS, tink key being used everywhere. This is probably fine, but something we definitely want to document and ensure users are aware of changing (and using a real KMS, key injection abstraction)